cmake/FindSGX.cmake

```diff
@@ -1,331 +1,14 @@
-# Copied from
```

Comment:
This file defined a bunch of environment variables and CMake functions that we do not need/use. At this stage, we just want to check that the Intel SDK is installed under `/opt/intel/sgxsdk`. Any different installation will probably break our build anyway.